corehq/apps/hqcase/templates/hqcase/explode_cases.html

```diff
@@ -7,7 +7,30 @@
              factor: ko.observable(''),
              user_id: ko.observable(''),
          });
-         $('#explode-user_id').select2();
+         $('#explode-user_id').select2({
+             ajax: {
+                 url: "{% url 'users_select2_options' domain %}",
+                 type: 'POST',
+                 datatype: 'json',
+                 quietMills: 250,
```

Comment:
nit: is 250 ms a very short duration? Should this be made like a second?

Reply:
that's what we use on other async select2 inputs